simple-auction/contracts/auction.ral

```diff
@@ -37,7 +37,7 @@ Contract Auction(
     let path = toByteVec!(from)
     let bidderContractId = subContractId!(path)
     if (!contractExists!(bidderContractId)) {
-      let expectedAlphAmount = amount + 1 alph
+      let expectedAlphAmount = amount + minimalContractDeposit!()
```

Comment:
Can we remove the contract deposit here?

Reply:
IIUC. we want `amount` to be something on top of the contract deposit as the initial amount for the sub contract, which wont be the case if we remove `minimalContractDeposit`.

Reply:
If the amount is greater than `minimalContractDeposit`, no deposit is needed, right?

Maybe we can just remove the contract deposit and check that `amount > minimalContractDeposit!()`.

It's an interesting case I didn't think of.

Reply:
But with the change, we could treat each bid as `amount` rather than `amount - minimalContractDeposit!`, right?

E.g. If I bid with 1 alph, then the amount is 1 alph. The contract deposit is covered by the 1 alph at the same time.
* When the bid succeeds, 1 alph is transferred and the contract is destroyed
* When the bid fails, 1 alph is returned back to the bidder

Reply:
Yes, that makes sense :+1: 
Fixed in the [last commit](https://github.com/alephium/ralph-example/pull/67/commits/f48a8db7cc7aec943a6c62963a8b079d7d36448f).